lambda/update/main.go

```diff
@@ -75,7 +77,11 @@ func (l *Lambda) HandleEvent(ctx context.Context, event events.APIGatewayProxyRe
 		return problem.Respond()
 	}
 
-	if err := l.store.Put(ctx, lpa); err != nil {
+	update.Uid = lpa.Uid
```

Comment:
Could we also generate a UUID here and save it to the update? Appreciate this wasn't in the original ticket, but it came up as part of the data doc that it would be helpful to have a unique identifier for each change.
```suggestion
	update.Id = uuid.NewString()
	update.Uid = lpa.Uid
```

Reply:
See fix